docs/source/about.rst

```diff
@@ -0,0 +1,33 @@
+.. _about-chapter:
+
+################
+About DI-Sensors
+################
+
+**************************
+Who are we and what we do.
+**************************
+
+.. the image size was extracted from the image manually
+.. without the image size, the scale cannot be set
+.. image:: https://i.imgur.com/V1LUKyq.jpg
+   :scale: 100
+   :width: 1695
+   :height: 532
+
+`Dexter Industries`_ is an American educational robotics company that develops robot kits that make programming accessible for everyone.
+
+********************************
+What's this documentation about.
+********************************
+
+This documentation is all about the sensors that `Dexter Industries`_ produces and maintains - that's where the **DI** acronym comes from. All the source code for these sensors has been written in Python.
+Within this documentation, you will find instructions on:
+
+   * How to get started with the DI-Sensors - in general it refers to how to install them on your Raspberry Pi.
+   * How to get going with the examples found in our repository. In the `DI-Sensors repository`_ that we maintain and care about, you can find all the source code for our sensors and example programs.
```

Comment:
"care about" is not appropriate. You care about a cat, a sibling, but not really code.  I would simply skip "that we maintain and care about"

Reply:
Yeah, I find it quite funny, now that I look back at what I wrote. I agree with you here.